examples/windows/onnx_ptq/genai_llm/README.md

```
@@ -60,6 +60,7 @@ The table below lists key command-line arguments of the ONNX PTQ example script.
 | `--layers_8bit` | Default: None | Use this option to Overrides default mixed quant strategy|
 | `--gather_quantize_axis` | Default: None | Use this option to enable INT4 quantization of Gather nodes - choose 0 or 1|
 | `--gather_block_size` | Default: 32 | Block-size for Gather node's INT4 quantization (when its enabled using gather_quantize_axis option)|
+| `--use_column_major` | Default: disabled | Apply column-major storage optimization for execution providers that need it. Only applicable for DQ-only quantization.|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the left for style "compact"

(MD060, table-column-style)